bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestSnapshot.java

```diff
@@ -190,6 +193,117 @@ public void testListTableSnapshotWithSingleArgs() throws Exception {
     Assert.assertEquals(1, listTableSnapshotsSize(Pattern.compile(tableName.getNameAsString())));
   }
 
+  @Test
+  public void testSnapshotTableWithNullAndEmptyString() {
+    Exception actualError = null;
+    try {
+      snapshot("test-snapshot", null);
+      Assert.fail("listTableSnapshots with null tableName should fail");
+    } catch (Exception ex) {
+      actualError = ex;
+    }
+    assertNotNull(actualError);
+    actualError = null;
+
+    try {
+      snapshot(null, tableName);
+    } catch (Exception ex) {
+      actualError = ex;
+    }
+    assertNotNull(actualError);
```

Comment:
can we make this a more specific assertion, like error message or exception type?

Reply:
Thanks for the review, I have updated assertions with exception-type for most of them.

Have left assertion for those methods which throws different exception with HBase1 & HBase2 implementation(eg. `Admin#snapshot(String, TableName)`)